### PR TITLE
adds adhesive a chemical that is to be used in future crafting prs

### PR DESCRIPTION
**Pull request by boodaliboo**
adhesive currently does nothing but i want to use it in future prs
craftable with (1 hydrogen,1 oxygen, 1 carbon)
#### Changelog

:cl:  
rscadd: Added a new chem that currently does nothing but will do something in the future
/:cl:

### CODE REVIEW

**Comment by Hopekz** [comment]
Bad game design to add a useless item unless its skub
This should be added with whatever it's used with.

**Comment by ToGWtF** [comment]
This would be the third time we've added duct tape/glue

**Comment by boodaliboo** [comment]
> This would be the third time we've added duct tape/glue

i havent seen them added and wanted this to be avaiable for other people to use

**Comment by Marmio64** [comment]
it'd be cool if you could use adhesive as makeshift nodrops, once applied you cant drop it normally, but you can still slowly tear it off your hand flesh or have someone else do it for you, but you'll take 30 brute damage and fall over from the fucking AGONY of tearing all of your skin off on your palm.

**Comment by ghost** [comment]
Adding glue yet a-fucking-gain I see

**Review by JamieD1 — APPROVED** [review]

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1768,3 +1768,10 @@
 	color = "#020202"
 	taste_description = "bananas"
 	can_synth = TRUE
+
+/datum/reagent/adhesive
+	name = "Adhesive"
+	description = "A Sticky substance that used to be made out of horses but is now creatable with chemicals"
```
Comment (Ktlwjec1) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: description = "A sticky substance that used to be made out of horses, but is now created with chemicals."

Comment (ThatLing) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: @Ktlwjec 

image

**Comment by ThatLing** [comment]
What's the point of adding this before adding anything that uses it?

**Comment by boodaliboo** [comment]
> What's the point of adding this before adding anything that uses it?
so others can use it

**Review by JamieD1 — APPROVED** [review]
Checks out

**Comment by SomeguyManperson** [comment]
nobody is gonna think "hm this random chemical that just exists without any practical application is perfect for my new idea" they'll just make a thing that works for their idea

**Comment by alexkar598** [comment]
No point in this existing innit

**Pull request closed without merging**